revenue_app/views.py

```diff
@@ -1,6 +1,6 @@
 from django.contrib.auth.mixins import LoginRequiredMixin
 from django.views.generic import TemplateView
-from .utils import get_organizer_list
+from .utils import get_organizer_list, get_organizer_event_list
```

Comment:
I think this needs to have parenthesis, like
`from .utils import (
    get_organizer_list,
    get_organizer_event_list,
)`

Reply:
ok!